From 60376cb2e64dc043dce660da823a737e5f44e577 Mon Sep 17 00:00:00 2001
From: Mike Gerdts <mike.gerdts@joyent.com>
Date: Fri, 23 Feb 2018 19:50:28 +0000
Subject: [PATCH] OS-6689 illumos-extra.manifest not regenerated when
 projects/illumos-extra/manifest changes Reviewed by: Jerry Jelinek
 <jerry.jelinek@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8b79c2f5..00098ebb 100644
--- a/Makefile
+++ b/Makefile
@@ -206,7 +206,8 @@ $(MCPROTO)/illumos.mancheck.conf: projects/illumos/mancheck.conf | $(MCPROTO)
 $(BOOT_MPROTO)/illumos.manifest: projects/illumos/manifest | $(BOOT_MPROTO)
 	cp projects/illumos/boot.manifest $(BOOT_MPROTO)/illumos.manifest
 
-$(MPROTO)/illumos-extra.manifest: 1-extra-stamp | $(MPROTO)
+$(MPROTO)/illumos-extra.manifest: 1-extra-stamp \
+    projects/illumos-extra/manifest | $(MPROTO)
 	gmake DESTDIR=$(MPROTO) DESTNAME=illumos-extra.manifest \
 	    -C projects/illumos-extra manifest; \
 
-- 
2.21.0

